components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -192,6 +198,26 @@ var cancelBackupCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(0),
 }
 
+var integrityBackupCmd = &cobra.Command{
+	Use:   "integrity COMMAND",
+	Short: "Chef Automate shared object integrity",
+}
+
+var integrityBackupShowCmd = &cobra.Command{
+	Use:   "show",
+	Short: "show the shared object integrity metadata",
+	Long:  "Show the shared object integrity metadata",
+	RunE:  runBackupIntegrityShowCmd,
+	Args:  cobra.ExactArgs(0),
+}
+
+var integrityBackupValidateCmd = &cobra.Command{
+	Use:   "validate [ID IDN]",
+	Short: "validate the shared object integrity",
+	Long:  "Validate the shared object integrity. Arguments are assumed to be backup ID's that you wish to validate, if left empty it will validate all snapshots",
```

Comment:
ID's -> IDs

[nit] The long usage message reads as a run-on sentence to me, so I'd probably use a semicolon instead of a comma after "validate".